tests/test_ilastik/conftest.py

```diff
@@ -7,3 +7,16 @@
 def data_path():
     conf_path = pathlib.Path(__file__).parent
     return conf_path / "data"
+
+
+def pytest_addoption(parser):
+    parser.addoption("--tiktorch_executable", help="tiktorch executable to use for integration tests")
```

Comment:
```suggestion
    parser.addoption("--tiktorch-executable", help="tiktorch executable to use for integration tests")
```
I know, it's then not consistent with argument of ilastik executable, but I think everyone from the devs expects this weird convention only from ilastik.